examples/cluster/plot_face_segmentation.py

```diff
@@ -26,26 +26,30 @@
 
 import numpy as np
 import scipy as sp
+from scipy.ndimage.filters import gaussian_filter
 import matplotlib.pyplot as plt
+from skimage.transform import rescale
 
 from sklearn.feature_extraction import image
 from sklearn.cluster import spectral_clustering
-from sklearn.externals._pilutil import imresize
 
 
 # load the raccoon face as a numpy array
 try:  # SciPy >= 0.16 have face in misc
     from scipy.misc import face
-    face = face(gray=True)
+    orig_face = face(gray=True)
 except ImportError:
-    face = sp.face(gray=True)
+    orig_face = sp.face(gray=True)
```

Comment:
to avoid using the `/ 255`, let's use make the following

```python
from skimage import img_as_float64

orig_face = img_as_float64(sp.face(gray=True))
```